packages/app/config/setup-entries.js

```diff
@@ -0,0 +1,42 @@
+const AssetsPlugin = require('assets-webpack-plugin');
+const merge = require('webpack-merge');
+const path = require('path');
+const paths = require('./paths');
+
+module.exports = config => {
+  const isProd = config.mode === 'production';
+  const shared = {
+    plugins: [
+      new AssetsPlugin({
```

Comment:
**Why do we need `AssetsPlugin`?**

See the MR description for more details.

This is a nice plugin which provides a JSON file that can map an entrypoint name to the actual file path that webpack outputs to. So, at runtime I can query for `assets.json` and look for the `sandpack-service-worker` entry to get the hashed file path.

Reply:
@CompuIves I think I remember going down this path, but finding some issue with it. But I think you're right! It **should** also do the job. I'm going to take a look 👀

Sorry for the late reply + hope you had a great new year / happy holidays!